src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/ConfirmRegulatedStandardController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Roatp.CourseManagement.Application.Standard.Queries;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure.Authorization;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.Standards;
+using System;
+using System.Threading.Tasks;
+
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Controllers
+{
+    [Authorize(Policy = nameof(PolicyNames.HasProviderAccount))]
+    public class ConfirmRegulatedStandardController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<EditCourseContactDetailsController> _logger;
+
+        public ConfirmRegulatedStandardController(IMediator mediator, ILogger<EditCourseContactDetailsController> logger)
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        [Route("{ukprn}/standards/{larsCode}/confirm-regulated-standard", Name = RouteNames.ConfirmRegulatedStandard)]
+        [HttpGet]
+        public async Task<IActionResult> ConfirmRegulatedStandard([FromRoute] int larsCode)
+        {
+            var ukprn = Ukprn;
+            _logger.LogInformation("Getting Course details for ukprn {ukprn} LarsCode {larsCode}", ukprn, larsCode);
+
+            var result = await _mediator.Send(new GetStandardDetailsQuery(ukprn, larsCode));
+
+            if (result == null || result?.StandardDetails == null)
+            {
+                _logger.LogError("Standard details not found for ukprn {ukprn} and larscode {larsCode}", ukprn, larsCode);
```

Comment:
You can create the message as a variable, and inject that variable into the LogError and Exception message